Formula/grokj2k.rb

```diff
@@ -26,13 +26,46 @@ class Grokj2k < Formula
   depends_on "libtiff"
   depends_on "little-cms2"
 
+  uses_from_macos "perl"
+  uses_from_macos "zlib"
+
+  on_macos do
+    depends_on "llvm" => :build if DevelopmentTools.clang_build_version == 1205
+  end
+
+  on_linux do
+    depends_on "gcc"
+  end
+
+  # GNU compiler version must be at least 10.0
+  # https://github.com/GrokImageCompression/grok/blob/master/INSTALL.md#compilers
+  fails_with gcc: "5"
+  fails_with gcc: "6"
+  fails_with gcc: "7"
+  fails_with gcc: "8"
+  fails_with gcc: "9"
```

Comment:
I think we can just do
```ruby
fails_with gcc: "9"
```

Reply:
I'll give it a try. Any recommendation on disabling exact clang? I added some lines all over to use LLVM instead to bypass Big Sur failure.

The formula failed on Apple Clang 12.0.5. This could be due to CI macOS Big Sur and XCode 12.5 update (Clang 12.0.0 --> 12.0.5), in which case Apple Clang 12.0.0 may actually work and in future > 12.0.5 may work, so I only wanted to fail on the specific version.

Reply:
Tried locally and didn't work. Seems to be specific to `gcc` version.

Reply:
@Bo98 is just doing `fails_with gcc: "9"` supposed to work here?